app/controllers/quizzes_controller.rb

```diff
@@ -41,8 +48,13 @@ def update
     end
   end
 
-  def check
+  def submit
     @quiz = Quiz.find(params[:id])
+    @user = User.first
+    @quiz_session = QuizSession.find_or_create_by(user: @user, quiz: @quiz)
```

Comment:
aici cred ca tb doar find cu session in_progress - ca session-ul e creat in show